src/components/ImageCard.tsx

```diff
@@ -21,7 +21,20 @@ const Description = styled.p`
   opacity: 0.8;
 `
 
-const ImageCard = ({ image, title, description, children, className }) => (
+export interface IProps {
+  image: string
+  title: string
+  description: string
+  className?: string
```

Comment:
We should pass alts to images explicitly :-)

Reply:
Yeah good idea, just added it